main.py

```diff
@@ -179,6 +179,10 @@ def trace_step(self, switch, entries, stored_flows):
             return None
 
         endpoint = find_endpoint(switch, port)
+        if endpoint is None:
+            log.warning("Interface not found")
```

Comment:
It's nice to have this log.warning here, but can you augment this? 

I mean when this ever happens, as it is, there's no information about which endpoint exactly, so it won't be immediately evident. Maybe we can have something like `f"Port {port} not found on switch {switch}"`?

Reply:
Thank you. Done!